contracts/eth/nearbridge/test/block_validators_v1_testnet.json

```diff
@@ -0,0 +1 @@
":"V1","account_id":"denny55.pool.f863973.m0","public_key":"ed25519:HHWD9y9gDzJjerKK8nxFkj3M3abXnQN7dQNevkKob8Cz","stake":"473978648319678259370867423422"},{"validator_stake_struct_version":"V1","account_id":"waynewayner.pool.f863973.m0","public_key":"ed25519:67fmk7fGcJF3SCuLptn1AtFaHeMFEX7A9mRacbkXaQoY","stake":"454067111142423517708146322354"},{"validator_stake_struct_version":"V1","account_id":"grassets.pool.f863973.m0","public_key":"ed25519:3S4967Dt1VeeKrwBdTTR5tFEUFSwh17hEFLATRmtUNYV","stake":"478178360387048275124610237011"},{"validator_stake_struct_version":"V1","account_id":"bloxrox.pool.f863973.m0","public_key":"ed25519:4ueWbLmGD7rxr2vcmwkTzs62qEWJt2Ndiyp5SLpnuCk3","stake":"473176089916916164684166646624"},{"validator_stake_struct_version":"V1","account_id":"bit2bit.pool.f863973.m0","public_key":"ed25519:CWThBEvdw2rvHScy6fD2nPuMoUTZ2sqVub2Xg4k1QB3U","stake":"477019547320668910880880201702"},{"validator_stake_struct_version":"V1","account_id":"stgr.pool.f863973.m0","public_key":"ed25519:84BuxAQ5vKZoWpJKQphuq6EiNVDsJ9Rok65dKQwfLZPt","stake":"472790190941473051926089657077"},{"validator_stake_struct_version":"V1","account_id":"leadnode.pool.f863973.m0","public_key":"ed25519:CdP6CBFETfWYzrEedmpeqkR6rsJNeT22oUFn2mEDGk5i","stake":"489053119473383704850265372706"},{"validator_stake_struct_version":"V1","account_id":"tayang.pool.f863973.m0","public_key":"ed25519:G9XWX55MfWEpT84ckcsJxVTKeZK4WqBGJX3xVpnPB5vv","stake":"461013250995447404407585046471"},{"validator_stake_struct_version":"V1","account_id":"pandateam.pool.f863973.m0","public_key":"ed25519:7n426KJocZpJ5496UHp6onwYqWyt5xuiAyzvTGwCQLTN","stake":"535054749408103884504027930007"},{"validator_stake_struct_version":"V1","account_id":"aurora.pool.f863973.m0","public_key":"ed25519:9c7mczZpNzJz98V1sDeGybfD4gMybP4JKHotH8RrrHTm","stake":"546282177598469747960331624999"},{"validator_stake_struct_version":"V1","account_id":"fortress.pool.f863973.m0","public_key":"ed25519:veXPT5KHyEgwmDqV41iFX9oZ3rH1ELUwqWtocAA7WC8","stake":"440120917627900576755211817069"},{"validator_stake_struct_version":"V1","account_id":"blockngine.pool.f863973.m0","public_key":"ed25519:CZrTtCP6XkkxWtr3ATnXE8FL6bcG5cHcxfmdRgN7Lm7m","stake":"485047708849773266720539666408"},{"validator_stake_struct_version":"V1","account_id":"shardlabs.pool.f863973.m0","public_key":"ed25519:DxmhGQZ6oqdxw7qGBvzLuBzE6XQjEh67hk5tt66vhLqL","stake":"480621524322218101752022303236"},{"validator_stake_struct_version":"V1","account_id":"stakingpower.pool.f863973.m0","public_key":"ed25519:HsXvTEyjhmitcdK1Uk6do6rzRqC7U3vtDk5CGCcHUrEn","stake":"471871567154659041926854422200"},{"validator_stake_struct_version":"V1","account_id":"lavenderfive.pool.f863973.m0","public_key":"ed25519:AzwAiLDqprZKpDjhsH7dfyvFdfSasmPTjuJUAHfX1Pg4","stake":"445374847495594702756596983569"},{"validator_stake_struct_version":"V1","account_id":"infiniteloop.pool.f863973.m0","public_key":"ed25519:2fbiLqksH5viWXYoteyfKP9qQawkRKw4YogRFcvG3Z7f","stake":"488482402962925655210618680533"}]}
```

Comment:
Does this need to be all one line? Makes it a pain to read.

Reply:
This is test data. I don't think it's intended to read it :) I don't think you can verify anything just by reading it.